cluster/manifests/stackset-controller/01-stackset-crd.yaml

```diff
@@ -536,29 +576,37 @@ spec:
                           minReplicas:
                             description: minReplicas is the lower limit for the number
                               of replicas to which the autoscaler can scale down.
+                              It defaults to 1 pod.
                             format: int32
                             type: integer
                         required:
                         - maxReplicas
                         - metrics
                         type: object
-{{- if eq .Cluster.ConfigItems.stackset_legacy_hpa_field_crd_enabled "true" }}
+# {{ if eq .Cluster.ConfigItems.stackset_legacy_hpa_field_crd_enabled "true" }}
```

Comment:
Template yaml-commented here

Reply:
why comment not remove?

Reply:
We need this conditional block to remove part of the CRD when it is false. I put it into yaml comment such that whole file is a valid yaml (see PR description). It is still evaluated because CLM template rendered does not care about yaml comments and treats the file just as plain text.